data/json/items/gun/223.json

```
@@ -536,8 +536,8 @@
     "id": "sig552",
     "copy-from": "rifle_auto",
     "type": "GUN",
-    "name": { "str": "SIG 552" },
-    "description": "A compact selective fire automatic rifle designed for the Swiss military.  It features a three-round burst mode and an integrated folding stock.",
+    "name": { "str": "SIG 552 SWAT" },
+    "description": "A select fire rifle originally made for the Swiss military and redesigned for American specifications.  This one is the shortened version, made with SWAT teams and vehicle crewmen in mind.  Unlike it's European original it features a STANAG magwell.",
```

Oh wait nevermind, typo
```suggestion
    "description": "A select fire rifle originally made for the Swiss military and redesigned for American specifications.  This is the shortened version, made with SWAT teams and vehicle crewmen in mind.  Unlike the European original it features a STANAG magwell.",
```